packages/data-helper/src/hooks/useNodeInfo/useNodeInfo.js

```diff
@@ -4,11 +4,15 @@ import {getQuery} from './useNodeInfo.gql-queries';
 import {useDeepCompareMemoize} from '../useDeepCompareMemo';
 import {getEncodedPermissionName} from '../../fragments/getPermissionFragment';
 import {getEncodedNodeTypeName} from '../../fragments/getIsNodeTypeFragment';
+import {useSchemaFields} from '../useSchemaFields';
 
 export const useNodeInfo = (variables, options, queryOptions) => {
+    let publicationInfoSchemaType = useSchemaFields({type: 'GqlPublicationInfo'});
+    variables.supportsExistsInLive = publicationInfoSchemaType && publicationInfoSchemaType.fields.find(field => field.name === 'existsInLive') !== undefined;
```

Comment:
Avoid changing the variables parameter, which is passed by the user. Instead, just pass the result of useSchemaFile as a new parameter of getQuery

Reply:
ok